bigbio/hub/hub_repos/sem_eval_2024_task_2/sem_eval_2024_task_2.py

```diff
@@ -169,35 +171,58 @@ def _info(self) -> datasets.DatasetInfo:
     def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
         """Returns SplitGenerators."""
 
-        urls = _URLS["train"]
-        data_dir = Path(dl_manager.download_and_extract(urls))
+        train_dev_dir = Path(dl_manager.download_and_extract(_URLS["train"]))
```

Comment:
My rewrite it as `paths = { split: dl_manager.download_and_extract[url] for split, url in _URLS.items()}` and then access the paths via dictionary lookups in the code below.

Reply:
Should be fixed with 450a5494e6fe2a968bb5dfede34193c095c41784